packages/material-ui/src/ListItemText/ListItemText.js

```diff
@@ -45,53 +96,58 @@ const ListItemText = React.forwardRef(function ListItemText(props, ref) {
   } = props;
   const { dense } = React.useContext(ListContext);
 
+  const styleProps = {
+    ...props,
+    inset,
+    primary: primaryProp,
+    secondary: secondaryProp,
+    dense,
+  };
+
+  const classes = useUtilityClasses(styleProps);
+
   let primary = primaryProp != null ? primaryProp : children;
   if (primary != null && primary.type !== Typography && !disableTypography) {
     primary = (
-      <Typography
+      <ListItemTextPrimary
```

Comment:
```suggestion
      <Typography
```

No need for new component as there are no styles associated with it